src/TreePicker/test/TreeNodeSpec.tsx

```diff
@@ -1,60 +1,66 @@
 import React from 'react';
 import { Simulate } from 'react-dom/test-utils';
 import sinon from 'sinon';
-import { getDOMNode } from '@test/testUtils';
 import TreeNode from '../TreeNode';
+import { render } from '@testing-library/react';
 
 describe('TreePicker - TreeNode', () => {
   it('Should render tree node', () => {
-    const instance = getDOMNode(<TreeNode layer={0} visible nodeData={{}} />);
+    const { container } = render(<TreeNode layer={0} visible nodeData={{}} />);
 
-    assert.include(instance.className, 'rs-tree-node');
-    assert.include(instance.getAttribute('role'), 'treeitem');
+    expect(container.firstChild).to.have.class('rs-tree-node');
+    expect(container.firstChild).to.have.attr('role', 'treeitem');
   });
 
   it('Should call `onDragStart` callback', () => {
     const onDragStartSpy = sinon.spy();
-    const instance = getDOMNode(
+    const { container } = render(
       <TreeNode layer={0} onDragStart={onDragStartSpy} nodeData={1} visible />
     );
-
-    Simulate.dragStart(instance);
+    //eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+    const div = container.querySelector('div') as HTMLElement;
```

Comment:
This query returns the root element all the same. Use `container.firstChild` (also remove `eslint-disable` annotation then).